image/utils/error.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+import "fmt"
+
+type ImageNameOrIDNotFoundError struct {
+	name string
+}
+
+func (e *ImageNameOrIDNotFoundError) Error() string {
+	errinfo := fmt.Sprintf("failed to find imageName or imageId %s", e.name)
```

Comment:
return fast

Reply:
return fmt.Sprintf("failed to find imageName or imageId %s", e.name)